scripts/clean

```diff
@@ -3,12 +3,21 @@
 for f in packages/*; do
 	if [[ -f ${f}/package.yaml ]]; then
 		if [[ -z $CHART || $CHART == $(basename -- ${f}) ]]; then
-			if [[ $(cat ${f}/package.yaml | yq r - url) ]]; then
+			# Remove special changes from optional flags before checking if Git is dirty
+			providesGVR="$(yq r ${f}/package.yaml providesGVR)"
+			if [[ -n ${providesGVR} && -f ${f}/charts/Chart.yaml ]]; then
+				if [[ "$(yq r ${f}/charts/Chart.yaml 'annotations[catalog.cattle.io/provides-gvr]')" == "${providesGVR}" ]]; then
+					yq d -i ${f}/charts/Chart.yaml "annotations[catalog.cattle.io/provides-gvr]" 
+				fi
+			fi
```

Comment:
As mentioned elsewhere, let's just remove all this `providesGVR` stuff because its disconnected from CRD generation now.

Reply:
Not sure if we should just remove this providesGVR stuff since we'll have to modify the patches of all of the `Chart.yaml`s for each chart. With the changes added in the last commit (discussed in https://github.com/rancher/charts/pull/620#discussion_r485799851), there shouldn't be a need for removing this stuff at the moment since it will no longer break the build.